packages/url_launcher/url_launcher/README.md

```diff
@@ -113,4 +149,4 @@ By default, Android opens up a browser when handling URLs. You can pass
 If you do this for a URL of a page containing JavaScript, make sure to pass in
 `enableJavaScript: true`, or else the launch method will not work properly. On
 iOS, the default behavior is to open all web URLs within the app. Everything
-else is redirected to the app handler.
\ No newline at end of file
+else is redirected to the app handler.
```

Comment:
```suggestion
else is redirected to the app handler.

```

What is this change? Is this missing a new line at the end of the file or something? This wasn't present in the previous PR.

Reply:
Not sure why this happened. I was using a merge tool to cherry pick changes from my deleted branch into the new fork. Thanks for the catch.